include/tvm/target/target_kind.h

```diff
@@ -102,7 +102,7 @@ class TargetKindNode : public Object {
 
   void VisitAttrs(AttrVisitor* v) {
     v->Visit("name", &name);
-    v->Visit("device_type", &device_type);
+    v->Visit("default_device_type", &default_device_type);
```

Comment:
this would be a convenient time to update a file e.g. tests/release/PRERELEASE_NOTES.md to indicate an "api change." what do you think about creating that file?

cc @driazati @tqchen @Mousius

Reply:
sounds like a good idea that would make generating coherent release notes a lot easier (at least to highlight big changes). line items like this should be enough:

```
* #<pr number>: <description>
* #12509: this does X changes API A from Y to Z
```

Reply:
I think it's a good idea to create the `PRERELEASE.md`, the one question I have is why under `tests`? It doesn't seem related to testing 🤔

Reply:
that's where we have other release scripts (for now: https://github.com/apache/tvm/tree/main/tests/scripts/release and https://discuss.tvm.apache.org/t/establishing-a-home-for-tvm-dev-tools/13212), but we could also just put it at the top level